plugin/loadbalance/README.md

```diff
@@ -19,7 +19,52 @@ implementations (like glibc) are particular about that.
 loadbalance [POLICY]
 ~~~
 
-* **POLICY** is how to balance. The default, and only option, is "round_robin".
+* **POLICY** is how to balance. Available options are "round_robin" and "weighted_round_robin". The default is "round_robin".
+
+The "round_robin" strategy randomizes the order of  A, AAAA, and MX records applying a uniform probability distribution.
+
+The "weighted_round_robin" policy assigns weight values to IPs to control the relative likelihood of particular IPs to be returned as the first
```

Comment:
I think just `weighted` should suffice for the name.

Reply:
Done.

Reply:
Yes, I agree. It's done.